src/proto/mgmt/control.proto

```diff
@@ -26,20 +26,28 @@ package mgmt;
 
 import "common.proto";
 import "storage.proto";
+import "pool.proto";
 import "features.proto";
 import "srv.proto";
 
 // This is the main service definition for communications involving the
-// gRPC management server. In order to add RPCs associated with your
+// gRPC management server. In order to add RPCs associated with new
 // service add them in the ManagementService definition below.
 service MgmtControl {
-	rpc ScanStorage(ScanStorageParams) returns (ScanStorageResp) {};
-	rpc FormatStorage(FormatStorageParams) returns (stream FormatStorageResp) {};
-	rpc UpdateStorage(UpdateStorageParams) returns (stream UpdateStorageResp) {};
-	rpc BurninStorage(BurninStorageParams) returns (stream BurninStorageResp) {};
-	rpc KillRank(DaosRank) returns (DaosResponse) {}; // Kill server of rank
-	rpc FetchFioConfigPaths(EmptyParams) returns (stream FilePath) {};
-	rpc GetFeature(FeatureName) returns (Feature) {};
-	rpc ListAllFeatures(EmptyParams) returns (stream Feature) {};
-	rpc ListFeatures(Category) returns (stream Feature) {};
+	// Retrieve details of nonvolatile storage devices on server
+	rpc ScanStorage(ScanStorageReq) returns (ScanStorageResp) {};
+	// Format nonvolatile storage devices for use with DAOS
+	rpc FormatStorage(FormatStorageReq) returns (stream FormatStorageResp) {};
+	// Update nonvolatile storage device firmware
+	rpc UpdateStorage(UpdateStorageReq) returns (stream UpdateStorageResp) {};
+	// Perform burn-in testing to verify nonvolatile storage devices
+	rpc BurninStorage(BurninStorageReq) returns (stream BurninStorageResp) {};
+	// Fetch FIO configuration file specifying burn-in jobs/workloads
+	rpc FetchFioConfigPaths(EmptyReq) returns (stream FilePath) {};
+	// Create a DAOS pool allocated across a number of ranks
+	rpc CreatePool(CreatePoolReq) returns (CreatePoolResp) {};
```

Comment:
CreatePool should belong to Management Service rather than Management Control Service.

Reply:
thanks, will amend. KillRank also? even if so, as it currently is not implemented in iosrv will leave KR where it is for the moment